### DISCUSSION TITLE
【求助】幫一下 谁带个路

### DISCUSSION THREAD

**Discussion by I-m-Miss-Lucy-s-dog** [share121/inter-knot#2838] [其他]
![下載](https://github.com/user-attachments/assets/aa69b25b-2189-4fdb-b094-c70487a26040)

去莱姆尼安空洞做委托时被一只地精以骸一棒敲昏 醒来就在這
还看到一个长得跟竹竿一样但没有以太核的以骸
有人知道这啥地方吗
来个人拿萝卜把我带出来

**Comment by awa1233** [comment]
这是哪个空洞,怎么没见过

**Reply by I-m-Miss-Lucy-s-dog** [reply]
莱姆尼安空洞那里

**Comment by yssxrc** [comment]
以骸是来给你带路的<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >

**Reply by I-m-Miss-Lucy-s-dog** [reply]
你说的没啥用 他貌似只想杀了我 好在我跑得快

**Comment by btazsq** [comment]
你现在看到的其实是种幻觉，你只要回忆一下童年时期的记忆就能解除这种幻觉了

**Comment by panda114514667** [comment]
那你完了，看看有什么家产挂悬赏吧

**Comment by Klein124284420** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by M1aoW** [comment]
![6569fmg8](https://github.com/user-attachments/assets/5c5f0825-c796-4657-8309-4623d9952649)
好巧兄弟，我在巴莱大厦空洞内部想见识网上说的关于双子的传说，那手萝卜可是我花高价买的，没想到居然碰到一个超大以骸，在逃跑过程中很不巧的踩空了楼梯，直接摔入了这里，想来绳网上求助居然发现了和我一样倒霉的人，目前还没有看见以骸，兄弟你还活着吗？

**Comment by shengwang999** [comment]
这个委托我接了！

**Comment by GanlinSama** [comment]
最近怎么这么多人掉后室里去了？

**Reply by bangbu01** [reply]
可能前厅出错误了

**Comment by haxyyx** [comment]
这里可能是未知的伴生空洞，我没有在治安局的档案看到这些，自求多福吧
